ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -738,6 +738,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-sdn-workers-rhcos-rhel8
+- as: aws-ipi-hypershift-install-test
+  cron: '@weekly'
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_FILTERS: HyperShiftINSTALL
+      TEST_SCENRAIOS: Hypershift
+    test:
+    - ref: openshift-extended-test-longduration
```

Comment:
`openshift-extended-test-longduration` might not the one you actually want. Please double confirm.

Reply:
yes, it's about installation test for hypershift